src/htdocs/storm/index.php

```diff
@@ -0,0 +1,92 @@
+<?php
+if (!isset($TEMPLATE)) {
+  $TITLE = 'Magnetic Disturbance Summaries';
+  $HEAD = '<link rel="stylesheet" href="index.css"/>';
+  $NAVIGATION = true;
+  include 'template.inc.php';
+}
+?>
+
+<div class="column">
+  <table>
+    <thead>
+      <tr>
+        <th>Date Interval</th>
+        <th>Name</th>
+        <th>Peak Dst</th>
+        <th>Map</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th>2015-06-21 16:45:00Z  to  2015-06-22 18:30:00Z</th>
+        <th><a href="/storm/storm23.php">Event: 2015-06-23</a></th>
+        <th>-195</th>
+        <th><a href="http://geomag.usgs.gov/map/#storm/23">Map</a></th>
```

Comment:
These types of links can be absolute paths (which are relative to the site and will inherit the protocol from the current page). For example: `/map/#storm#23`